src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -122,7 +128,17 @@ CancellationToken ct
         Uri uri = BuildUri(baseUrl, pagePath, language);
 
         bool displayFooter = _pdfGeneratorSettings.DisplayFooter;
-        string? footerContent = displayFooter ? GetFooterContent(instance, textResource) : null;
+
+        string? footerContent = null;
+
+        if (isPreview == true)
```

Comment:
Should we not check `displayFooter` when generating preview? I'd imagine that might pop us as unexpected behavior from users

Reply:
This is according to spec: https://github.com/Altinn/app-frontend-react/issues/1804 :)